File: c++/uva10055.cpp
#include <iostream>
#include <cstdlib> // 用於 abs() 函數

using namespace std;

int main() {
    long long armyHashmat, armyOpponent;
    
    // 當還有輸入時，持續執行
    while (cin >> armyHashmat >> armyOpponent) {
        // 計算並輸出兩軍隊數量的絕對差
        cout << abs(armyHashmat - armyOpponent) << endl;
    }
    
    return 0;
}

File: java/uva10055.java
import java.util.Scanner;

public class uva10055 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        while (scanner.hasNextLong()) {
            // 讀取兩個軍隊的數量
            long armyHashmat = scanner.nextLong();
            long armyOpponent = scanner.nextLong();
            
            // 計算並輸出差的絕對值
            System.out.println(Math.abs(armyHashmat - armyOpponent));
        }
        
        // 關閉Scanner以釋放資源
        scanner.close();
    }
}